src/html-conventions-lib/class-html-conventions.xsl

```diff
@@ -54,17 +54,16 @@
             </h2>
             <section>
                 <xsl:if test="boolean($classConventions)">
-                    <h3>Unmet class conventions</h3>
                     <section>
```

Comment:
no need for a new section anymore. merge the attributes and the class errors into a single section.

Reply:
good suggestion.